File: Programmers/src/Javascript/영어가싫어요.js
function solution(numbers) {
    var answer = "";
    let num_list = ["zero","one","two","three","four","five","six","seven","eight","nine"];
    let num = [0,1,2,3,4,5,6,7,8,9];
    
    for(let i=0; i<num_list.length; i++){
        numbers = numbers.replaceAll(num_list[i], num[i]);
    }
    
    answer = Number(numbers);
    return answer;
}

File: Programmers/src/Java/Lv0/영어가 싫어요.java
package Java.Lv0;

class Solution {
    public long solution(String numbers) {
        String[] num = {"zero","one","two","three","four","five","six","seven","eight","nine"};
        
        for(int i=0; i<num.length; i++) {
            numbers = numbers.replaceAll(num[i], String.valueOf(i));
        }
        
        return Long.parseLong(numbers);
    }
}